dev-guide/cluster-fleet-evaluation.md

```diff
@@ -0,0 +1,181 @@
+---
+title: cluster-fleet-evaluation
+authors:
+  - "@aravindhp"
+reviewers:
+  - "@openshift/openshift-patch-managers"
+approvers:
+  - "@openshift/openshift-patch-managers"
+creation-date: 2022-08-16
+last-updated: 2022-08-16
+---
```

Comment:
I didn't expect to see header metadata here, but it adds a nice bit of history so I like it.